Configs/Config.Tests.ps1

```diff
@@ -93,7 +93,7 @@ Process {
         }
 
         It 'Contains proper settings for .vds' {
-            $VMKeys = 'linkproto|linkoperation|mtu'
+            $VDSKeys = 'linkproto|linkoperation|mtu'
             $config.vds.Keys | Should Match $VMKeys
```

Comment:
Now you have to update this because you listened to me :innocent:

Thanks for the + tip! I need to go track down a "GitHub on Day 2" walkthrough.